src/stacked_borrows.rs

```diff
@@ -471,46 +463,60 @@ impl Stacks {
     ) -> (Self, Tag) {
         let (tag, perm) = match kind {
             MemoryKind::Stack =>
-                // New unique borrow. This tag is not accessible by the program,
-                // so it will only ever be used when using the local directly (i.e.,
-                // not through a pointer). That is, whenever we directly write to a local, this will pop
-                // everything else off the stack, invalidating all previous pointers,
-                // and in particular, *all* raw pointers.
-                (Tag::Tagged(extra.borrow_mut().new_ptr()), Permission::Unique),
+            // New unique borrow. This tag is not accessible by the program,
+            // so it will only ever be used when using the local directly (i.e.,
+            // not through a pointer). That is, whenever we directly write to a local, this will pop
+            // everything else off the stack, invalidating all previous pointers,
+            // and in particular, *all* raw pointers.
+            {
+                (
+                    Tag::Tagged(extra.borrow_mut().new_ptr()),
+                    Permission::Unique,
+                )
+            }
```

Comment:
Woah, this is just horrible formatting.^^ *And* it moved comment and code apart.